src/network/mod.rs

```diff
@@ -53,18 +53,12 @@ pub mod stream_reader;
 pub enum Error {
```

Comment:
In 5c6d36928958af467719df2f6318dae43646f548. 
Don't know whether we should use `io::Error` now that this only has one variant.

Reply:
Yes, good suggestion, my bad for not thinking of this originally, thanks. https://github.com/rust-bitcoin/rust-bitcoin/pull/1032